specs/inheritance.yml

```diff
@@ -0,0 +1,192 @@
+overview: |
```

Comment:
Is this yaml file added for the purpose of documentation?

Reply:
(This is also related to your other question on `.json` minification). Specs in the [official repository](https://github.com/mustache/spec) are written in YAML format as the reference, and then [programmatically](https://github.com/mustache/spec/blob/master/Rakefile#L12-L27) rendered into JSON to be usable from people outside the YAML cult.

The existing spec files in the `specs/` directory of ocaml-mustache exactly follow this format: both versions wered copied over, the JSON version is used by the tool and the YAML version is read by humans when a test fails. The import I did here follows the same approach.

I think that it is okay to keep the current (non-ideal) workflow. In any case, we should keep the YAML files around, because they are the reference format, so we want people to be able to "diff" them against the current version of the spec to see if we missed some changes. (The spec repository currently appears to be unmaintained, but oh well.)

We could probably add some tooling of our side to re-render the JSON files from the YAML file with human-friendly pretty-printing. (One fun aspect of it is that the JSON specs from the upstream repo are [currently out of date](https://github.com/mustache/spec/pull/113), so doing this refresh would in fact change slightly the tests we are running. Who knows, they may even catch a mistake and start failing.) I would propose to wait until there is some `yaml` library used here (possibly from #45)  to do this.

Reply:
Alright. We should note this in a ticket then. Using a yaml library for the tests sounds fine as well.